host.js

```diff
@@ -54,7 +54,10 @@ const originRe = /^(http:\/\/|https:\/\/)[^/]+/i;
 const tryGetOrigin = (url) => {
 	if (url && url.indexOf('//') === 0) {
 		url = window.location.protocol + url;
+	} else if (url && url.indexOf('/d2l/') === 0) {
+		url = window.location.origin + url;
```

Comment:
This should be able to just be something more like `return window.location.origin` rather than running it through the regex, but :shrug:

Reply:
Could you add to the host origin tests, https://github.com/Brightspace/ifrau/blob/master/test/host.test.js#L40